src/ios/CDVCamera.m

```diff
@@ -59,6 +59,59 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 @implementation CDVPictureOptions
 
+/*
+ API:
+ Camera:
+ - CameraPopoverHandle:
```

Comment:
This was meant as a quick way to check the API (and most of all the options) while developing. I can get rid of it (but it can be a good way to know if we're in sync with the docs).